pebbles/static/partials/account.html

```diff
@@ -20,8 +20,10 @@
         <tr>
             <td>Name</td>
             <td>Role</td>
-            <td>Details</td>
```

Comment:
We are not removing "Details". As another "Join Code" is inserted before "Details", it is showing as deleted and reinserted below ....